packages/gallery/src/components/helpers/renderCursorController.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { CursorController } from './mouseCursorPosition';
+
+export function ArrowsContainer({
+  hideRightArrow,
+  hideLeftArrow,
+  mouseCursorEnabled,
+  children,
+}) {
+  if (mouseCursorEnabled) {
+    const styleForMouseCursor = {
+      justifyContent: hideLeftArrow
+        ? 'flex-end'
+        : hideRightArrow
+        ? 'flex-start'
+        : 'space-between',
+    };
+    return (
+      <div className="mouse-cursor" style={{ ...styleForMouseCursor }}>
+        {children}
+      </div>
+    );
+  }
+  return <>{children}</>;
+}
+
+export function renderArrowButtonWithCursorController(props) {
+  return (
+    <CursorController>
+      {({ containerRef, position, isMouseEnter }) =>
+        renderArrowButton({
+          cursor: {
+            containerRef,
+            position,
+            isMouseEnter,
+          },
+          ...props,
+        })
+      }
+    </CursorController>
+  );
+}
+
+export function renderArrowButton({
```

Comment:
why is this related to cursor controller (i.e why in this file and not an arrow related file?